romancal/regtest/test_dark_current.py

```diff
@@ -0,0 +1,82 @@
+"""Regression test for the Dark current subtraction step."""
+import pytest
+
+from romancal.stpipe import RomanStep
+from .regtestdata import compare_asdf
+
+
+@pytest.mark.bigdata
+def test_dark_current_subtraction_step(rtdata, ignore_asdf_paths):
+    """ Function to run and compare Dark Current subtraction files. Note: This
+        should include tests for overrides etc. """
```

Comment:
Are the overrides in the reference / output files, or something in the code?

Reply:
strun allows you to override many of the defaults and we should test them. You can see some of the options with
strun --verbose  roman_elp -h

and some of the overrides,
153:  --steps.dq_init.override_mask
180:  --steps.saturation.override_saturation
207:  --steps.linearity.override_linearity
236:  --steps.dark_current.override_dark
275:  --steps.jump.override_gain
277:  --steps.jump.override_readnoise
309:  --steps.rampfit.override_readnoise
311:  --steps.rampfit.override_gain
363:  --steps.flatfield.override_flat

Reply:
Okay, then the note should probably be deleted from here, since this test doesn't test overrides.. that test is the XFAIL at the bottom.

Reply:
Deleted.